foundations-macros/src/span_fn.rs

```diff
@@ -41,12 +41,19 @@ impl Parse for SpanName {
 struct Options {
     #[darling(default = "Options::default_crate_path")]
     crate_path: Path,
+
+    #[darling(default = "Options::default_async_local")]
+    async_local: bool,
```

Comment:
is this commonly known to carry the meaning/semantics you want to?

or would it be better to call it `not_send`?

Reply:
As far as I can tell, yes:
- https://docs.rs/tokio/latest/tokio/task/struct.LocalSet.html (search for "local futures")
- https://docs.rs/futures/latest/futures/future/type.LocalBoxFuture.html